### PR TITLE
Create BLorient12114A

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record according to the description in S. Strelcyns catalogue.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient12114A.xml [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+                                    <locus target="#128r"/>
+                                    <desc type="OwnershipNote">Scribbles, including the name of the owner 
+                                        <persName role="owner" ref="PRS14362KidanaWald">Kidāna Wald</persName>.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding>
+                                <decoNote xml:id="b1">The binding is badly damaged.</decoNote>
+                                <decoNote xml:id="b2" type="Boards"><material key="wood"/>Wooden boards.</decoNote>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient12114A.xml:65]: "The binding is badly damaged" would be better under "Condition"

File: LondonBritishLibrary/orient/BLorient12114A.xml [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+                        </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="p1_i1">
+                            <locus from="1r" to="163vb"/>
+                            <title ref="LIT2701Dawit"/>
+                            <msItem xml:id="p1_i1.1">
+                                <locus from="1r" to="128v"/>
+                                <title ref="LIT2000Mazmur"/>
+                                <note>The chapters bear the same title:
+                                    <q xml:lang="gez"><title ref="LIT2000Mazmur">መዝሙር፡ ዘዳዊት።</title></q></note>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient12114A.xml:88]: I think he does not mean "chapters"; he means the so-called traditional title of each psalm which exist in several versions. Here, each of them has the title Mazmur za-dawit.

File: LondonBritishLibrary/orient/BLorient12114A.xml [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="18"><locus from="1r" to="149v"/></layout>
+                                <layout columns="2" writtenLines="18"><locus from="150r" to="163v"/></layout>
+                            </layoutDesc>    
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Rather careful handwriting.</desc>
+                                <date notBefore="1600" notAfter="1800" resp="PRS8999Strelcyn"/>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient12114A.xml:222]: Do we have any guidelines as to how we write the dating? Strelcyn might probably mean "17th or 18th century" (what I  personally indicate 17th/18th century). The dating embracing the span of 200 years is a little odd (in his catalogue mostly without arguments anyway).

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient12114A.xml [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">163+1</measure>
+                                </extent>
+                                <collation>One bifolium after <ref target="#141"/>, one leaf after <ref target="#150"/>, probably two leaves after
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient12114A.xml:44]: I think this belongs to condition. Collation is about the composition of the quires.

File: LondonBritishLibrary/orient/BLorient12114A.xml [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+                            <title ref="LIT2701Dawit"/>
+                            <msItem xml:id="p1_i1.1">
+                                <locus from="1r" to="128v"/>
+                                <title ref="LIT2000Mazmur"/>
+                                <note>The chapters bear the same title:
+                                    <q xml:lang="gez"><title ref="LIT2000Mazmur">መዝሙር፡ ዘዳዊት።</title></q></note>
+                            </msItem>
+                            <msItem xml:id="p1_i1.2">
+                                <locus from="128v" to="149v"/>
+                                <title ref="LIT1233Cantic"/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient12114A.xml:92]: `type="incomplete"`

File: LondonBritishLibrary/orient/BLorient12114A.xml [hunk 6]
```diff
@@ -0,0 +1,352 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient12114" xml:lang="en" type="mss">
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient12114A.xml:5]: `xml:id` must be the same as the name of the file

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

**Comment by CarstenHoffmannMarburg** [comment]
@DenisNosnitsin1970 
I think, I made the changes, that you requested. Can this branch be merged?

**Pull request merged**